### PR TITLE
linkagg: Add initial integration test

### PR DESCRIPTION
**Pull request by tyll**

### CODE REVIEW

File: tests/README-integration-tests.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Integration Tests #
+
+The integration tests are written as Ansible playbooks targetted for all hosts.
+They follow the [Fedora's Standard Test
+Interface](https://fedoraproject.org/wiki/CI/Standard_Test_Interface) and can
+be run by the [Linux Sytem Roles Test
```
Comment (EdDev) [inline tests/README-integration-tests.md]: Tipo: System

File: tests/README-integration-tests.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Harness](https://github.com/linux-system-roles/test-harness). You can also run
+them directly, for example to run them against the [CentOS Cloud
```
Comment (EdDev) [inline tests/README-integration-tests.md]: I will use "steps/points" here, it is hard to follow in this long sentence.

**Review by EdDev — COMMENTED** [review]
Small nits.

File: tests/README-integration-tests.md (hunk 1 above)
Comment (tyll) [reply tests/README-integration-tests.md]: fixed

File: tests/README-integration-tests.md (hunk 2 above)
Comment (tyll) [reply tests/README-integration-tests.md]: I broke it up into two sentences.

**Review by EdDev — APPROVED** [review]
